content/blog/2021-11-30-international-rse-survey.md

```diff
@@ -0,0 +1,33 @@
++++
+title = "Take the 2021 international RSE survey: help us improve the RSE community"
+
+[extra]
+
+author = "Patric Holmvall, Richard Darst" 
++++
+
+
+## Take the survey
+
+Do you develop research codes, or generally work with numerics or data analysis? Perhaps you even feel more connected to the technology side: programming, software, data, etc., rather than the writing of papers? Although this work is crucial to the research, the modern research environment seldom fully recognizes this work, in contrast to other methodology development. Take part in the [international Research Software Engineer (RSE) survey](https://softwaresaved.limequery.com/386272?lang=en) to help us understand how we can improve the situation. Your current position or job title does not matter, only that you feel connected to software, data, computation, or similar technology.
```

Comment:
```suggestion
Do you develop research codes, or generally work with research software or data analysis? Perhaps you even feel more connected to the technology side: programming, software, data, etc., rather than the writing of papers? Although this work is crucial to the research, the modern research environment seldom fully recognizes this work, in contrast to other methodology development. Take part in the [international Research Software Engineer (RSE) survey](https://softwaresaved.limequery.com/386272?lang=en) to help us understand how we can improve the situation. Your current position or job title does not matter, only that you feel connected to software, data, computation, or similar technology.
```

Reply:
Reason for change: one can work with numerics without doing RSE work and there is a lot of RSE work that is not numerics. But perhaps I am unsure about the term "numerics".

Reply:
No I think you're right!